Makefile.in

```diff
@@ -19,7 +19,7 @@ distclean	: clean
 	rm -f Makefile
 
 install		:
-	install -p -D --target-directory=$(bindir) $(PROGNAME)
```

Comment:
`--target-directory` is unfortunately not recognized by install in Busybox (e.g. Alpine).